sections/futures/Trade/DepositMarginModal.tsx

```diff
@@ -29,6 +29,9 @@ type DepositMarginModalProps = {
 	market: string | null;
 };
 
+const PLACEHOLDER = '$0.00';
+const MIN_DEPOSIT_AMOUNT = wei('100');
```

Comment:
I think with the SCCP that was just merged, this can change to this now:
```suggestion
const MIN_DEPOSIT_AMOUNT = wei('50');
```

Reply:
@brossetti1 Just one small change, but everything looks good.